client/state/data-layer/wpcom-http/pipeline/retry-on-failure/index.jsx

```diff
@@ -0,0 +1,108 @@
+/**
+ * External dependencies
+ */
+import {
+	get,
+	head,
+	sortBy,
+	toPairs,
+} from 'lodash';
+
+const requestQueue = new Map();
+
+export const buildKey = ( { path, apiNamespace, apiVersion, query } ) => JSON.stringify( [
```

Comment:
yep - thought about it - hadn't done it yet in case that's not how we want it to end up